src/helics/core/BrokerBase.hpp

```diff
@@ -135,8 +137,8 @@ class BrokerBase {
     /** specify that outgoing connection should use json serialization */
     bool useJsonSerialization{false};
     bool enable_profiling{false};  //!< indicator that profiling is enabled
-    decltype(std::chrono::steady_clock::now())
-        errorTimeStart;  //!< time when the error condition started related to the errorDelay
+    /// time when the error condition started related to the errorDelay
```

Comment:
```suggestion
    /// time when the error condition started relative to the errorDelay
```
 Relative to, or just related?

```suggestion
    /// time when the error condition started; related to the errorDelay
```

Reply:
indication that this variable is connected to errorDelay,  vs relative